demo-app/src/pages/default/index.jsx

```diff
@@ -20,17 +21,16 @@ const hostedConfig = {
 // eslint-disable-next-line no-unused-vars
 const pmtilesConfig = {
   format: 'pmtiles',
-  // TODO: Add PMtiles file to CDN and update URL
-  url: 'http://localhost:8080/planet/{z}/{x}/{y}.mvt',
+  url: import.meta.env.VITE_TILE_LAYER_URL,
```

Comment:
When I open the netlify preview, I can't see the maps. Do we need to add this env variable to Netlify?

Reply:
I've added already, just after the original preview was built. I have re-deployed and verified: https://deploy-preview-91--whimsical-moxie-6e43a0.netlify.app/

Might have been cached when you first viewed, but should be coming through now.

Reply:
Works!